client/my-sites/posts/post-controls.jsx

```diff
@@ -144,14 +145,14 @@ const getAvailableControls = props => {
 
 	if ( controls.more.length ) {
 		controls.main.push( {
-			className: 'more',
+			className: 'more' + ( current === 'more' ? ' is-active' : '' ),
 			icon: 'ellipsis',
 			onClick: onShowMore,
 			text: translate( 'More' ),
 		} );
 
 		controls.more.push( {
-			className: 'back',
+			className: 'back' + ( current === 'back' ? ' is-active' : '' ),
```

Comment:
I was reticent to do it because it could be unclear but yes ... it's better. Let me update the code.